### ISSUE TITLE
Fade text in? (Suggestion)

### ISSUE THREAD

**Issue by Lolagatorade** [ViperX7/Alpaca-Turbo#1]
It would be nice to have the text fade in as it is processing? Unless multiple scripts are being written and the best is chosen. Otherwise waiting on a blank loading screen is significantly less engaging than watching the words appear.

**Comment by ViperX7** [comment]
Done

**Comment by Lolagatorade** [comment]
> Done  The interface now shows streaming responses

Seems to behave. Ime running yesterdays build. Although it did have a tendency to stall with responses and the messages would not show, the cpu usage went to zero. So when a new request was added the response that did not appear would come up.
